charts/tfy-prometheus-config/values.yaml

```diff
@@ -248,9 +438,10 @@ serviceMonitors:
     jobLabel: app.kubernetes.io/instance
     ## @param serviceMonitors.prometheus.endpoints [array] Endpoints for prometheus
     endpoints:
-      - port: http-web
-        interval: 30s
-        path: /metrics
+      - path: /metrics
```

Comment:
Is the `reloader-web` required or part of the scrape config which was explicitly added ?

Reply:
it is part of the scrape config that was explicitly added from `selfMonitor: true`

Reply:
Let's not add that until we need it